### ISSUE TITLE
Documentation update suggestion for README.md in 0-bootstrap

### ISSUE THREAD

**Issue by marktomlinson1** [terraform-google-modules/terraform-example-foundation#611]
### TL;DR

Default GCP Cloud Shell version of Terraform is currently 1.0.x. Latest is 1.1. Secure Cloud Foundation only works with 0.13.x. Suggest providing instructions on how to install backlevel version of Terraform if running locally in step 0-bootstrap README.

### Expected behavior

Terraform scripts running locally in Cloud Shell execute without deprecation errors.

### Observed behavior

Scripts in future 1-org steps when running locally use syntax deprecated since Terraform 0.12 and do not run on default 1.0.x version in Cloud Shell. Resolved by installing backlevel 0.13 version into Cloud Shell before first steps executed. 

### Terraform Configuration

```hcl
terraform version
```

### Terraform Version

```sh
Terraform v0.13.0
```

### Additional information

I downloaded the older version from https://www.terraform.io/downloads and then overwrote /usr/bin/terraform in Cloud Shell with the downloaded version. Suspect there are better approaches including using tfswitch as /usr/bin changes are often overwritten. Running locally.

**Comment by rr4444** [comment]
Yup, this is very frustrating. I just killed my state backend with an up to date local client :(

**Comment by kvendingoldo** [comment]
btw. you can also use [tenv](https://github.com/tofuutils/tenv) that support Terraform as well as OpenTofu (and Terragrunt :) ) in one tool. It allow you to simplify version management and can do much more, than tfswitch.
